source/distributed/tria.cc

```diff
@@ -1823,6 +2357,24 @@ namespace parallel
           Assert(false, ExcInternalError());
         }
 
+      // load saved data, if any was stored
+      if (attached_count > 0)
+        {
+          data_transfer.load(parallel_forest, filename, attached_count);
+
+          data_transfer.unpack_cell_status(local_quadrant_cell_relations);
+
+          // the CellStatus of all stored cells should always be CELL_PERSIST.
+          for (auto &quad_cell_rel : local_quadrant_cell_relations)
```

Comment:
The variable is unused in Release mode so we need a `(void) quad_cell_rel` below. While being here, you can also make the variable `const`.